client/components/Dummy.jsx

```diff
@@ -1,13 +1,43 @@
 import React from 'react';
+import jQuery from 'jquery';
 import DummyChild from './DummyChild';
+import styled from 'styled-components';
```

Comment:
Since `styled-components` is an external library, it's better to group all the external libraries at the top, and then import the `DummyChild` so that way imports can be sorted & grouped in a way that it's easier to navigate.

[Reference](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md)